lib/nsfunc/application-navigation-api/resolve-url.js

```diff
@@ -20,9 +20,12 @@ exports.nlapiResolveURL = (type, identifier, id, displayMode) => {
 
     let db = global.$db;
     if (~['suitelet', 'restlet'].indexOf(type)) {
-        if (!identifier) throw nlapiCreateError('SSS_IDENTFIER_ARG_REQ');
+        if (!identifier) throw nlapiCreateError('SSS_MISSING_REQD_ARGUMENT');
```

Comment:
"INVALID_ID" is what netsuite actually throws. If docs say otherwise, docs are wrong.